sympy/printing/tests/test_latex.py

```diff
@@ -1,6 +1,6 @@
 from sympy import (
     Add, Abs, Chi, Ci, CosineTransform, Dict, Ei, Eq, FallingFactorial,
-    FiniteSet, Float, FourierTransform, Function, IndexedBase, Integral,
+    FiniteSet, Float, FourierTransform, Function,Indexed, IndexedBase, Integral,
```

Comment:
Please add a space after comma.

Reply:
@jksuom Sure sir I have updated the code as asked by you.